src/i18n/en.js

```diff
@@ -185,7 +186,7 @@ export default {
     list_subtitle: `You can synchronize the data manually whenever you want.`,
     list_title: `Synchronized List`,
     table_list: `List Name`,
-    table_subscribers_count: `Subscribers`,
+    table_subscribers_count: `Shopify customers`,
```

Comment:
What do you think about also changing the key? maybe:

```suggestion
    table_shopify_customers_count: `Shopify customers`,
```

Reply:
I thought that maybe in the future we will reuse this as subscribers, but ... YAGNI.